### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cwc-products**

### CODE REVIEW

**Comment by delraydavis** [comment]
Chris is this the area where we would offer feedback?

**Comment by lingyaintocareers** [comment]
@cwc-products , please resolve the conflict

**Comment by delraydavis** [comment]
@cwc-products this pull request needs to have it's merge issues addressed

**Comment by intoCareers** [comment]
@cwc-products - please resolve your conflicts

**Comment by papajeremy** [comment]
@cwc-products please fix conflicts before merge approval

**Comment by cwc-products** [comment]
Resolved conflicts, please review the code by following this link: https://github.com/cwc-products/simpleTwo/pull/11/files.

**Review by lingyaintocareers — APPROVED** [review]
fine

**Review by delraydavis — CHANGES_REQUESTED** [review]
please add some punctuation!!!!!!!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # simpleTwo
 
-Go Ducks! - Christopher
+Go Ducks! - Chris
```
Comment (intoCareers) [inline README.md]: Your name is Christopher

**Review by intoCareers — CHANGES_REQUESTED** [review]
See comment above

File: README.md (hunk 1 above)
Comment (cwc-products) [reply README.md]: No I want to go Chris.

**Review by delraydavis — APPROVED** [review]

**Review by intoCareers — APPROVED** [review]

**Pull request merged**